src/platforms/common/enriching-events/attachments/index.mdx

```diff
@@ -10,14 +10,13 @@ notSupported:
   - native.breakpad
   - native.crashpad
   - native.minidumps
-  - native.ue4
```

Comment:
cc @jan-auer, looking at comments this seems to have never been intended https://github.com/getsentry/sentry-docs/pull/2343